ui/apps/everest/.e2e/db-cluster/db-wizard/create-db-cluster/create-db-cluster.e2e.ts

```diff
@@ -335,4 +339,21 @@ test.describe('DB Cluster creation', () => {
 
     await deleteDbClusterFn(request, clusterName, namespace);
   });
+
+  test('Warning should appears for schedule with the same date and storage', async ({
+    page,
+  }) => {
+    await page.goto('/databases');
+    await page.getByTestId('add-db-cluster-button').click();
+
```

Comment:
Let's add this before `moveForward`:
```
await expect(page.getByTestId('toggle-button-group-input-db-type')).toBeVisible();
```

Because sometimes the E2E click on "next" is quicker than the DB engines loading, and we get flaky tests (button will be disabled until DB engines are shown)